web_config/webconfig.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python
```

Comment:
The `/usr/bin/env python3` sheband should be used to allow user changing python exec from its env.